gov_uk_dashboards/colours.py

```diff
@@ -37,3 +37,10 @@ class GovUKColours(Enum):
         "#11436e",
         "#092237",
     ]
+
+
+class ONSAccessibleColours(Enum):
```

Comment:
Can a docstring be added here?

Reply:
Oooh I like splitting them out into a new class.  Agree with Kate, would be nice to link to where the colours came from.